packages/eslint-plugin-turbo/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  roots: ["<rootDir>"],
+  transform: {
+    "^.+\\.tsx?$": "ts-jest",
+  },
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+  modulePathIgnorePatterns: ["<rootDir>/node_modules", "<rootDir>/dist"],
+  preset: "ts-jest",
+};
```

Comment:
Unrelated to this PR, but do we have any shared configs for common stuff like this across the repo?

Reply:
Nope. That's something that we'd like to do, but has not yet been priority.

Reply:
Yep, we should do this soon (as we're adding more packages...)